supportedBackends/google/batch/src/main/scala/cromwell/backend/google/batch/api/GcpBatchRequestFactoryImpl.scala

```diff
@@ -231,6 +231,7 @@ class GcpBatchRequestFactoryImpl()(implicit gcsTransferConfiguration: GcsTransfe
     val machineType = GcpBatchMachineConstraints.machineType(runtimeAttributes.memory,
                                                              runtimeAttributes.cpu,
                                                              cpuPlatformOption = runtimeAttributes.cpuPlatform,
+                                                             standardMachineTypeOption = runtimeAttributes.standardMachineType,
```

Comment:
What this PR is still missing is the necessary changes for the "standardMachineType" key to be recognized in the wdl runtime section.

Reply:
Resolved. I didn't realize I had added the key as StandardMachineType (uppercase).